src/providers/timer-storage/timer-storage.ts

```diff
@@ -52,7 +52,7 @@ export class TimerStorageProvider {
   public modifyTimer(singleTimer: SingleTimer) {
     return this.getAllTimers().then(timers => {
       timers[timers.findIndex(timer => timer.id === singleTimer.id)] = singleTimer;
-      return this.storage.set(TIMERS_KEY, timers);
+      return this.storage.set(TIMERS_KEY, timers).then(_ => this.triggerTimersChanged());
```

Comment:
Is "_" a standard? Wouldn't two brackets be the better option? 
Like: .then(()=> this.triggerTimersChange())

Reply:
Does the same but is one character shorter and thus often used:
https://stackoverflow.com/questions/41085189/using-underscore-variable-with-arrow-functions-in-es6-typescript